### PR TITLE
Fix typos in docs

### PR DESCRIPTION
**Pull request by dhutchis**

### CODE REVIEW

**Comment by dhutchis** [comment]
Could someone take a look at my changes to the getting started page here? 
https://github.com/dhutchis/myria-docs/commit/6402d34cdb6d5876bd266b834d107b6439e1f57f?short_path=e701188#diff-e701188abab5b493e5915f1270430909

The other changes are easy and simple fixes.

File: myriaMiddleware.markdown [hunk 1]
```diff
@@ -24,8 +24,8 @@ Motivated by these ideas, we are working to expand the University of Washington
 Myria is a hosted Big Data management and analytics service consisting of three components:
 
 * [MyriaX](https://github.com/uwescience/myria): A big-data execution engine emphasizing asynchronous iterative processing.
-* [MyriaMiddleware](https://github.com/uwescience/raco): A language translation layer, shared optimizer, and query execution coordinator supporting a) multiple input languages and b) multiple back-end systems, of which MyriaX is one.   We emphasize a new imperative, iterative,  language called MyriaQL.
-* [MyriaWeb](https://github.com/uwescience/myria-web): A web-based editor and IDE designed for direct use by analysts, with support for collaborative editing through a shared workspace, visual performance analysis and debugging, and interactive algorithm.
+* [MyriaMiddleware](https://github.com/uwescience/raco): A language translation layer, shared optimizer, and query execution coordinator supporting a) multiple input languages and b) multiple back-end systems, of which MyriaX is one.   We emphasize a new imperative, iterative,  language called [MyriaQL](http://myria.cs.washington.edu/docs/myriaql.html).
```
Comment (domoritz) [inline myriaMiddleware.markdown]: Can you use a realtive link here?

**Comment by dhutchis** [comment]
Yes, relative links work when the docs are deployed on the [UW website](http://myria.cs.washington.edu/docs/index.html). They will dangle when accessed through the Github.  

I wonder if the deployment process could convert links from `FILENAME.markdown` to `FILENAME.html`...

**Comment by jortiz16** [comment]
Other than the comment above, everything else looks great. Also if you happen to take a look at the Python or MyriaX page where we detail how to run queries against the service, any feedback/changes on those pages would be great as well.

**Comment by dhutchis** [comment]
Thanks Jennifer!

**Pull request merged**